### ISSUE TITLE
Could not spawn process for application /var/lib/redmine: The application encountered the following error: undefined method `skip_before_filter' for PushoverCallbacksController:Class Did you mean?  skip_before_action (NoMethodError)

### ISSUE THREAD

**Issue by romingun** [jkraemer/redmine_pushover#7]
Environment:
  Redmine version                4.0.3.stable
  Ruby version                   2.4.6-p354 (2019-04-01) [x86_64-linux]
  Rails version                  5.2.2.1
  Environment                    production
  Database adapter               Mysql2

**Comment by syats** [comment]
Getting this issue as well
with Redmine 4.0.3 official docker image, with a Postgres DB.

Help would be appreciated

```

redmine_1  | /usr/src/redmine/plugins/redmine_pushover/app/controllers/pushover_callbacks_controller.rb:4:in `<class:PushoverCallbacksController>': undefined method `skip_before_filter' for PushoverCallbacksController:Class (NoMethodError)
redmine_1  | Did you mean?  skip_before_action
redmine_1  | 	from /usr/src/redmine/plugins/redmine_pushover/app/controllers/pushover_callbacks_controller.rb:3:in `<top (required)>'
redmine_1  | 	from /usr/local/bundle/gems/activesupport-5.2.2.1/lib/active_support/dependencies.rb:291:in `require'
redmine_1  | 	from /usr/local/bundle/gems/activesupport-5.2.2.1/lib/active_support/dependencies.rb:291:in `block in require'

```

**Comment by jkraemer** [comment]
please try again with current master, this should be fixed now thanks to #6
